packages/braid-design-system/src/lib/components/RadioGroup/RadioGroup.tsx

```diff
@@ -22,6 +22,7 @@ export type RadioGroupBaseProps<Value = NonNullable<string | number>> =
     onChange: (event: FormEvent<HTMLInputElement>) => void;
     name?: string;
     size?: InlineFieldProps['size'];
+    tone?: Exclude<FieldGroupBaseProps['tone'], 'caution'>;
```

Comment:
Should we use `Extract` instead of `Exclude` here? If a new tone was added tomorrow, `RadioGroup` would automatically have it as one of its `tone`s. If the new tone was never intended to be supported by `RadioGroup`, then you'd have to do a breaking change to remove it from the prop type.

Unrelated to this change, but should [the docs](https://seek-oss.github.io/braid-design-system/components/RadioGroup#message-and-tone) mention `positive` the tone here since it's on the prop type?

Reply:
Yeah I like that point. Will update, and interesting catch on `positive` being missing too!